### PR TITLE
Bug/restrict driver permissions

### PR DESCRIPTION
**Pull request by Somiel**
- [x] Prevent drivers from updating orders data w/o manager approvement
- [x] Add "in_progress" status for orders
- [x] Make drivers available to see both active and in_progress orders
- [x] Refactor free drivers scope

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -86,9 +84,9 @@ def order_params
 
   def set_orders
     case current_user
-    when Admin || Manager then @q = Order.ransack(params[:q]); @orders = @q.result(distinct: true)
-    when Driver then @orders = Order.where(driver_id: current_user.id)
-    when Client then @orders = Order.where(client_id: current_user.id)
+    when Admin || Manager then @q = Order.ransack(params[:q]); @orders = @q.result.page(params[:page]).per(20)
+    when Driver then @orders = current_user.orders.active + current_user.orders.in_progress
```
Comment (murderface13) [inline app/controllers/orders_controller.rb]: why driver needs to see all in progress orders?
i thought about in progress order, assigned for that driver

Comment (Somiel) [inline app/controllers/orders_controller.rb]: he sees only orders assigned to himself
current_user.orders.active + current_user.orders.in_progress , kinda tailwhip))

Comment (murderface13) [inline app/controllers/orders_controller.rb]: ok, i see. sorry for that

**Comment by murderface13** [comment]
:+1:

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (Somiel) [inline app/controllers/orders_controller.rb]: np :smile:

**Pull request merged**